cbqueryx/queryrespreader.go

```diff
@@ -136,6 +136,9 @@ func (r *queryRespReader) parseError(errJson *queryErrorJson) *ServerError {
 		if strings.Contains(lowerMsg, "build already in progress") {
 			err = ErrBuildAlreadyInProgress
 		}
+		if strings.Contains(lowerMsg, "build index fails") {
```

Comment:
This message should probably include the 'will retry' text, since that's an important detail to how this error is used.